backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -46,7 +46,7 @@ public static IQueryable<Entity.Project> GenerateQuery(this PimsContext context,
             {
                 var statuses = context.Workflows.Where(w => w.Code == "ACCESS-DISPOSAL")
                     .SelectMany(w => w.Status).Select(x => x.StatusId).ToArray();
-                query = query.Where(p => statuses.Contains(p.StatusId) || p.Status.Code.Equals("SU"));
+                query = query.Where(p => statuses.Contains(p.StatusId) || p.Status.Code.Equals("RE"));
```

Comment:
I suspect this is incorrect.  I think this was being used to return projects for the Approval Requests.  It would be returning projects that are in the `ACCESS-DISPOSAL` workflow and those that were submitted `SU`.

Reply:
The old submitted status is now review, so this is only a change in naming. Are you saying this check was incorrect before my change?